### ISSUE TITLE
option to ignore multiple mods warnings on either a per mod or per character basis

### ISSUE THREAD

**Issue by Pyrageis** [Jorixon/JASM#278]
its not common but theres definitely times where two mods of one character can work together, its always bugged me seeing the icon there when I know its fine

**Comment by kaosjr** [comment]
I agree, I use splash art mods in Star Rail which don't conflict with skin mods and I prefer to have them in the same section as the character. I'd appreciate a game-wide toggle as well if possible.
